portfolio/src/main/webapp/index.html

```diff
@@ -7,14 +7,24 @@
     <title>My Portfolio</title>
   </head>
   <body>
+<<<<<<< HEAD
```

Comment:
Looks link a funky merge happened here. (There are merge marker here on line 10, and also on line 19 and 36). Please fix up index.html so that it looks completely the way you want it to and make sure to test it!

Reply:
Yes I did not notice that. I fixed it. Thanks a lot for the help.